Source/PyromaniacIsFun/NeedPyromania.cs

```diff
@@ -131,27 +131,27 @@ public float GainFromObservedFireInterval()
             {
                 var gainFromWildFire = numWildFire * Patcher.Settings.NeedPyromaniaGainPerWildFirePerDay;
                 gain += gainFromWildFire;
-                sb.AppendFormat("Saw wild fire x{0}: +{1:F0}% ~{2}\n",
-                    numWildFire, gainFromWildFire * 100,
-                    (PyromaniacUtility.ObservedWildFireDef.DurationTicks - minAgeWildFire).ToStringTicksToPeriod()
+                sb.AppendFormat("CF_PyromaniacIsFun_NeedPyrimania.SawWildFire".Translate(numWildFire, (gainFromWildFire * 100).ToString("F0"),
+                    (PyromaniacUtility.ObservedWildFireDef.DurationTicks - minAgeWildFire).ToStringTicksToPeriod())
+                    
                 );
             }
             if (numBurningPawn > 0)
             {
                 var gainFromBurningPawn = numBurningPawn * Patcher.Settings.NeedPyromaniaGainPerBurningPawnPerDay;
                 gain += gainFromBurningPawn;
-                sb.AppendFormat("Saw burning pawn x{0}: +{1:F0}% ~{2}\n",
-                    numBurningPawn, gainFromBurningPawn * 100,
-                    (PyromaniacUtility.ObservedBurningPawnDef.DurationTicks - minAgeBurningPawn).ToStringTicksToPeriod()
+                sb.AppendFormat("CF_PyromaniacIsFun_NeedPyrimania.SawBurningPawn".Translate(numBurningPawn, (gainFromBurningPawn * 100).ToString("F0"),
+                    (PyromaniacUtility.ObservedBurningPawnDef.DurationTicks - minAgeBurningPawn).ToStringTicksToPeriod())
+                    
                 );
```

Comment:
Suggest `sb.AppendLine` and remove `\n` from the string

Reply:
Applied